README.md

```diff
@@ -7,6 +7,22 @@
 
 Fabric React is a responsive, mobile-first collection of robust components designed to make it quick and simple for you to create web experiences using the Office Design Language.
 
+## Created with Fabric
+
+[<img align="left" height="40" src='./ghdocs/img/yammer.png'/>](https://www.yammer.com)
+[<img align="left" height="50" src='./ghdocs/img/outlook-350-150.png'/>](https://www.outlook.com)
+[<img height="50" src='./ghdocs/img/VS_rgb_Purple.png'/>](https://www.visualstudio.com/team-services/")

```

Comment:
super nit: consider alt tags for these image links

https://www.w3.org/TR/2013/CR-html5-20130806/embedded-content-0.html#alt

Reply:
You can probably do this in pure Markdown. I think GitHub runs Kramdown. If so https://kramdown.gettalong.org/quickref.html#inline-attributes should work. But that's a stylistic thing so doesn't bother me.. just sharing.

Alt tag would be useful for a11y reasons is all.

Glad to see partners surfaced! 🎉

Reply:
not a nit, totally valid. Will update.